File: CS.NET/FrmDataGrid.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;

namespace Softgroup
{
	public partial class FrmDataGrid
	{


		internal FrmDataGrid()
		{
			InitializeComponent();
		}
		private void FrmDataGrid_Load(object sender, System.EventArgs e)
		{

			//Sets NetResize Licenses Informations
			//With Me.NetResize1
			//    .LicenseName = "your license name or company name"
			//    .LicenseUser = "your license user name or email"
			//    .LicenseKey = "your license key"
			//End With

			LoadData();

		}

		private void LoadData()
		{

			string[] Data = new string[6];
			int Row = 0;
			int Col = 0;

			for (Row = 0; Row <= 10; Row++)
			{
				Col = 0;
				Data[Col] = "Row" + Row.ToString() + " Col" + Col.ToString();
				Col = 3;
				Data[Col] = "Row" + Row.ToString() + " Col" + Col.ToString();
				DataGridView1.Rows.Add(Data);
			}

		}

		private void NetResize1_AfterControlResize(System.Windows.Forms.Control oControl, bool bCtrlResized, bool bFontResized)
		{
			this.DataGridView1.AutoResizeColumns(DataGridViewAutoSizeColumnsMode.AllCells);
		}

	}
} //end of root namespace

File: VB.NET/FrmDataGrid.vb

Public Class FrmDataGrid


    Private Sub FrmDataGrid_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load

        'Sets NetResize Licenses Informations
        'With Me.NetResize1
        '    .LicenseName = "your license name or company name"
        '    .LicenseUser = "your license user name or email"
        '    .LicenseKey = "your license key"
        'End With

        LoadData()

    End Sub

    Private Sub LoadData()

        Dim Data(5) As String
        Dim Row As Integer = 0
        Dim Col As Integer = 0

        With DataGridView1
            For Row = 0 To 10
                Col = 0
                Data(Col) = "Row" & Row.ToString & " Col" & Col.ToString
                Col = 3
                Data(Col) = "Row" & Row.ToString & " Col" & Col.ToString
                .Rows.Add(Data)
            Next
        End With

    End Sub

    Private Sub NetResize1_AfterControlResize(ByVal oControl As System.Windows.Forms.Control, ByVal bCtrlResized As Boolean, ByVal bFontResized As Boolean)
        Me.DataGridView1.AutoResizeColumns(DataGridViewAutoSizeColumnsMode.AllCells)
    End Sub

End Class